config/example/tips.txt

```diff
@@ -12,78 +12,77 @@ You can grab someone by clicking on them with the grab intent, then upgrade the
-As the Head of Personnel, you are just as large a target as the Captain because of the potential power your ID and computer can hand out.
+As the Executive Officer, you are just as large a target as the Captain because of the potential power your ID and computer can hand out.
 As the Chaplain, your bible is also a container that can store small items.
 As the Chaplain, your null rod has a lot of functions: it can convert water into holy water, which has a few practical applications. The rod itself can be used against evil and supernatural elements aswell.
 As the Chaplain, you are much more likely to get a response by praying to the gods than most people. To boost your chances, make altars with colorful crayon runes, lit candles, and wire art.
 As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic seeds. These seeds can alternatively be ordered from cargo.
 As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
-As a Cook, any food you make will be much healthier than the junk food found in vendors. Having the crew routinely eating from you will provide minor buffs.
+As a Cook, any food you make will be much healthier than the junk food found in vendors. The crew will thank you for eating healthier, eventually.
 As the Bartender, the drinks you start with only give you the basics. If you want more advanced mixtures, look into working with chemistry, hydroponics, or even mining for things to grind up and throw in!
 As the Bartender, you can use a circular saw on your shotgun to make it easier to store.
 As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer or order another from Cargo.
 As a Janitor, mousetraps can be used to create bombs or booby-trap containers.
 As the Librarian, be sure to keep the shelves stocked and the library clean for the crew.
-As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
-As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!
-As the Quartermaster, be sure to check the manifests on crates you receive to make sure all the info is correct. If there's a mistake, stamp the manifest DENIED and send it back in a crate with the items untouched for a refund!
+As a Hangar Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
```

Comment:
We don't even like have MULEbots anymore, even on Aurora we didn't, I'm pretty sure. Can just delete this.

Reply:
There were some on the Aurora. I wasn't sure. But I will fix.